multi-tenancy-observability/README.md

```diff
@@ -0,0 +1,154 @@
+---
+creation_date: 2024-02-08
+issues:
+- https://github.com/giantswarm/roadmap/issues/2771
+owners:
+- https://github.com/orgs/giantswarm/teams/team-atlas
+state: review
+summary: How Giant Swarm manages multi-tenancy to isolate observability data per tenants (i.e. metrics, logs, traces).
+---
+
+# Multi-tenancy for observability
+
+## Glossary
+
+- `tenant`: a group of users who share a common access with specific privileges to observability data
```

Comment:
Which users? Of the customer, of Giant Swarm? Maybe provide examples.

Reply:
```suggestion
- `tenant`: a group of individuals (from our customers or Giant Swarm alike) who share a common access with specific privileges to observability data
```